src/js/01-nav.js

```diff
@@ -8,9 +8,13 @@
   const languageMap = {
     en: {
       title: 'Home',
+      current_ver: 'Current version',
+      previous_ver: 'Previous versions',
```

Comment:
@cheungaryk  What about putting currentVersion and previousVersion I think it makes more sense.

Reply:
it would make sense if the words are 'Previous version' instead of 'Previous version**s**'. With 1 being single and 1 being plural, they are quite different...